src/components/Badge/Badge.tsx

```diff
@@ -46,43 +43,54 @@ const badgeVariants = {
 };
 
 const badgeFontSizes = {
-  [BadgeSizes.md]: 11,
+  [BadgeSizes.md]: 14,
   [BadgeSizes.sm]: 9,
```

Comment:
@julesbarr could you please kill the `sm` size? It was used in the platform when the ticket was created, but it's gone now.